### PR TITLE
Update policy about promoting CI jobs to informing/blocking dashboards

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this:

/kind documentation

#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/sig-release/issues/1611

#### Special notes for your reviewer:

PTAL @kubernetes/sig-release-leads @kubernetes/release-managers @kubernetes/release-team

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
nice update, thanks
/hold for more eyes
/lgtm

File: release-blocking-jobs.md [hunk 1]
```diff
@@ -118,26 +118,36 @@ based on the following considerations:
...
+
+1. Ensure approval from the job OWNERS, SIG Release Tech Leads or Chairs as well
+   as:
+
+   - the current Release Team Lead, CI Signal Lead and Release Manager if the
+     intention is to change a `sig-release-master-*` dashboard
+   - the Release Managers if the intention is to change a `sig-release-1.x-*`
+     dashboard
+
+1. Ensure that lazy consensus is reached in the issue by setting a deadline.
```
Comment (ameukam) [inline release-blocking-jobs.md:150]: Do we need to be explicit about how long we need to wait before we decide on the consensus?

Comment (ameukam) [reply release-blocking-jobs.md:150]: Yeah. we should give a few (4 ?) weeks for the community to review them. Also we want to get consensus before the code freeze or test freeze of each milestone ?

Comment (saschagrunert) [reply release-blocking-jobs.md]: Good, I changed the paragraph to add the additional information.

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

**Pull request merged**